newrelic/structures_newrelic_alert_condition.go

```diff
@@ -22,10 +22,10 @@ func expandAlertCondition(d *schema.ResourceData) (*alerts.Condition, error) {
 	condition.Terms = expandAlertConditionTerms(d.Get("term").(*schema.Set).List())
 
 	if violationCloseTimer, ok := d.GetOk("violation_close_timer"); ok {
-		if condition.Scope == "instance" {
+		if condition.Type == "apm_app_metric" && condition.Scope == "instance" {
```

Comment:
I don't think this fixes my issue, when condition.Type == "apm_jvm_metric", the if at L25 is false, so we go to the else and we still get an error.
The error should not be raised when condition.Type == "apm_jvm_metric"

Reply:
If you change this the if statement to:
if condition.Type != "apm_app_metric" || (condition.Type == "apm_app_metric" && condition.Scope == "instance")

Reply:
@gjkamstra You are 💯 correct, I have another commit queued up that looks pretty similar.  We had to provision some Java apps reporting to our test account to put integration tests in place, so this is taking a little longer than expected.  Thanks for your patience while we shave our yaks! 🐢